doxia-integration-tools/src/main/java/org/apache/maven/doxia/tools/DefaultSiteTool.java

```diff
@@ -911,6 +924,27 @@ private File resolveSiteDescriptor(
         return siteDescriptor;
     }
 
+    // TODO Remove this transient method when everyone has migrated to Maven Site Plugin 4.0.0+
+    private void deletePseudoSiteDescriptorMarkerFile(RepositorySystemSession repoSession, ArtifactRequest request) {
+        LocalRepository localRepository = repoSession.getLocalRepository();
+        LocalRepositoryManager lrm = repoSession.getLocalRepositoryManager();
+
+        String relativeLocalArtifactPath = lrm.getPathForLocalArtifact(request.getArtifact());
+        Path localArtifactPath = localRepository.getBasedir().toPath().resolve(relativeLocalArtifactPath);
+
+        try {
+            if (Files.exists(localArtifactPath) && Files.size(localArtifactPath) == 0L) {
+                LOGGER.debug(
+                        "Deleting 0-byte pseudo marker file for artifact '{}' at '{}'",
+                        request.getArtifact(),
+                        localArtifactPath);
+                Files.delete(localArtifactPath);
+            }
+        } catch (IOException e) {
+            // swallow
```

Comment:
Eek; at least log at debug level?

Reply:
What wouls be the improvement if I log this here?

Reply:
Perhaps a permissions error occurred during deletion? Perhaps acquiring the file size failed in some way? If things like this happen, I would certainly want to know about them. I've rarely seen a case where swallowing an exception is a good thing to do, but I'm not deeply familiar with the practices of Maven plugins and such.

Reply:
I agree with you.